src/main/java/frc/robot/RobotContainer.java

```diff
@@ -32,24 +35,22 @@
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
   public static DrivetrainSubsystem m_drivetrainSubsystem = new DrivetrainSubsystem();
-  private final ClimberSubsystem m_climberSubsystem = new ClimberSubsystem();
-  // private final AutoCommand m_autoCommand = new
-  // AutoCommand(m_drivetrainSubsystem);
-  private final IndexerSubsystem m_indexerSubsystem = new IndexerSubsystem();
-  private final IntakeSubsystem m_intakeSubsystem = new IntakeSubsystem();
+  public static ClimberSubsystem m_climberSubsystem = new ClimberSubsystem();
+  public static IndexerSubsystem m_indexerSubsystem = new IndexerSubsystem();
+  public static IntakeSubsystem m_intakeSubsystem = new IntakeSubsystem();
   public static GyroSubsystem m_gyroSubsystem = new GyroSubsystem();
-  private final ShooterSubsystem m_shooterSubsystem = new ShooterSubsystem();
-  
-  public Joystick m_leftStick = new Joystick(Constants.initialJoystickPort);
-  public Joystick m_rightStick = new Joystick(Constants.secondaryJoystickPort);
-  public XboxController m_xbox = new XboxController(Constants.xboxPort);
-
-  public Joystick LeftStick;
-  public Joystick RightStick;
+  public static ShooterSubsystem m_shooterSubsystem = new ShooterSubsystem();
+  public static LimelightSubsystem m_LimelightSubsystem = new LimelightSubsystem();
 
   // XBOX Contoller Defs (For intake and Climber)
-
-  // different Autos
+  public static XboxController m_driverXbox = new XboxController(0);
+  public static XboxController m_xbox = new XboxController(1);
+  public Trigger driverLeftTrigger = new Trigger(() -> IntakeSubsystem.getLeftTriggerActive());
+  public Trigger driverRightTrigger = new Trigger(() -> IntakeSubsystem.getRightTriggerActive());
```

Comment:
Why use IntakeSubsystem here instead of the class member m_intakeSubsystem?  The same goes for the ClimberSubsystem instead of m_climberSubsystem in the lines below.